Tinifier.Core/Infrastructure/SolutionExtensions.cs

```diff
@@ -48,11 +49,9 @@ public static byte[] ReadFully(Stream input)
         /// <returns></returns>
         public static string GetAbsoluteUrl(Media uMedia)
         {
-            var url = uMedia.GetUrl("Image", null);
-            UmbracoHelper umbHelper = Umbraco.Web.Composing.Current.UmbracoHelper;
-            var content = umbHelper.Media(uMedia.Id);
-            var imagerUrl = content?.Url;
-            return imagerUrl;
```

Comment:
This works for existing media items but not for new ones, that have not yet been added to the cache.